src/main/java/FileReadWrite.java

```diff
@@ -0,0 +1,45 @@
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.List;
+
+/**
+ * <p>The FileReadWrite class handles the writing to and reading
+ * from saved file with static methods.</p>
+ */
+public class FileReadWrite {
+    private static String home = System.getProperty("user.home");
+    private static java.nio.file.Path path = java.nio.file.Paths.get(home, "iP", "SavedData");
+    private static String fullFilePath = path + "/SavedData.txt";
+    private static boolean directoryExists = java.nio.file.Files.exists(path);
+
+    public static void writeToFile(String textToAdd) throws IOException {
+
+        // if file not existed, create new then add
+        if (!directoryExists) {
```

Comment:
this might be a problem since you initialise it statically; if the user deletes the folder for whatever reason in between saves, it might exist at first but disappear later